parser/sql/statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/mysql/ddl/MySQLDropEventStatement.java

```diff
@@ -17,12 +17,11 @@
 
 package org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.ddl;
 
-import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
-import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.DDLStatement;
+import org.apache.shardingsphere.sql.parser.sql.common.statement.ddl.DropEventTriggerStatement;
 import org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.MySQLStatement;
 
 /**
  * MySQL drop event statement.
  */
-public final class MySQLDropEventStatement extends AbstractSQLStatement implements DDLStatement, MySQLStatement {
+public final class MySQLDropEventStatement extends DropEventTriggerStatement implements MySQLStatement {
```

Comment:
Why change AbstractSQLStatement to DropEventTriggerStatement? Usually only when multiple dialects contain this statement will we consider extracting the common superclass.

Reply:
Thank you for your guidance. I understand :blush: